.github/actions/setup-environment/action.yml

```diff
@@ -316,12 +316,17 @@ runs:
           echo "gp2-app-url=https://$PR_NUMBER.gp2.asap.science" >> $GITHUB_OUTPUT
           echo "gp2-api-url=https://api-$PR_NUMBER.gp2.asap.science" >> $GITHUB_OUTPUT
 
-          if [ "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]; then
+          if [[ "$CRN_CREATE_ANALYTICS_ALGOLIA_INDEX" = true && "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]]; then
```

Comment:
I was unsure about the syntax but it works (I mean, it doesn't give an error and it reaches the content inside the `if` statement) as we can see here https://github.com/yldio/asap-hub/actions/runs/11146299276/job/30977928232#step:3:98
<img width="911" alt="Screenshot 2024-10-02 at 12 01 02" src="https://github.com/user-attachments/assets/f27bbf23-1047-4752-96fd-9db1ce9f3d4b">

I've cancelled the run with the two labels (`crn-create-environment` and `crn-create-analytics-algolia-index`) so it won't create a new analytics algolia index